workspace/__lib__/xod/core/cast-to-string(errcode)/patch.cpp

```diff
@@ -0,0 +1,28 @@
+
+#pragma XOD dirtieness disable
+
+struct State {
+    char str[5]; // Exxx\0
+    CStringView view;
+    State() : view(str) { }
+};
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
+    auto state = getState(ctx);
+    auto errcode = getValue<input_IN>(ctx);
+
+    char* str = state->str;
+
+    do
+        *str++ = (char)(48 + (errcode % 10));
+    while (errcode /= 10);
+
+    *str++ = 'E';
+    *str = '\0';
```

Comment:
Perhaps the standard [`itoa`](http://www.cplusplus.com/reference/cstdlib/itoa/) will be more straightforward here

Reply:
I don't think it's available on every platform

Reply:
OK, let it be